docs/feature-flags/feature-flags-logs.md

```diff
@@ -48,11 +48,11 @@ The table below indicates which attributes should be added to the
 
 The event name MUST be `feature_flag`.
 
-This event describes feature flag evaluation representation on a Log Record.
+This event describes feature flag evaluation.
 
 | Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
 |---|---|---|---|---|---|
-| [`feature_flag.key`](/docs/attributes-registry/feature-flag.md) | string | The unique identifier of the feature flag. | `logo-color` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`feature_flag.key`](/docs/attributes-registry/feature-flag.md) | string | The unique identifier of the feature flag. | `logo-color` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Is the change from recommended to required here intentional? 🤔

Reply:
It actually is.

We currently have two almost identical feature flag definitions (one for events, one for logs) and they are only different in the requirement level on this attribute. I stole the fix from https://github.com/open-telemetry/semantic-conventions/pull/1440 


@dyladan and @open-telemetry/semconv-feature-flag-approvers to confirm

Reply:
I wasn't sure when to have an events.yaml and when to have a logs.yaml. Is there anywhere documentation which describes when to use which one?

Reply:
I *think* it should be event.yaml, but really the file name is not enforced or regulated.

The convention itself should be for events because of

> * A semantic convention needs to be defined. We do not define semantic
  conventions for LogRecords that are not Events.

https://github.com/open-telemetry/opentelemetry-specification/blob/8548dfc199a76504d573880c4a3fe5a0e483357b/specification/logs/event-api.md?plain=1#L75

Let me create an issue so we document our stance on span events vs events vs logs in semantic conventions

Reply:
Created https://github.com/open-telemetry/semantic-conventions/issues/1526 and renamed file to `events.yaml`